src/zksync/artifacts/mod.rs

```diff
@@ -196,6 +196,8 @@ pub struct Settings {
     /// A flag indicating whether to forcibly switch to the EVM legacy assembly pipeline.
     #[serde(default)]
     pub force_evmla: bool,
+
```

Comment:
Docs might be useful on what it does (and that it's specific to solc). We can use the info from the PR description.